content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst

```diff
@@ -0,0 +1,257 @@
+   area.
+
+.. _inventory/shipping_receiving/docks:
+
+Dock locations
+--------------
+
+Each loading dock that delivery trucks load products from must have a dedicated dock location set.
+To create or configure dock locations, go to :menuselection:`Inventory app --> Configuration -->
+Locations`.
+
+Choose the desired location, and on the :guilabel:`Location` page, in the :guilabel:`Additional
+Information` section, tick the :guilabel:`Is a Dock Location` checkbox.
+
+.. figure:: dispatch/dock-location.png
+  :alt: Location configuration.
+
+  Location configuration page with :guilabel:`Is a Dock Location` checkbox ticked.
+
+Build loads
+===========
+
+After completing setup, batches of orders can be grouped together to load onto a delivery truck. To
+do that, orders must be :ref:`assigned to the carrier
+<inventory/shipping_receiving/assign-carrier>`. Then, group delivery orders together to :ref:`create
+a batch <inventory/shipping_receiving/create-batch>`. Finally, :ref:`configure the batch form
+<inventory/shipping_receiving/batch-form>` correctly.
+
+Since this article is about a specific use case, the details about each picking methods can be
+further explored in their dedicated articles.
+
+.. seealso::
+   - :doc:`../picking_methods/batch`
+   - :doc:`Wave picking <../picking_methods/wave>`
+   - :doc:`../picking_methods/cluster`
+
+To group products, go to the :menuselection:`Inventory app --> Operations --> Deliveries`, which
+reveals a list of outgoing deliveries.
+
+.. _inventory/shipping_receiving/assign-carrier:
+
+Carrier assignment
+------------------
+
+Reveal the :guilabel:`Carrier` column if it is not visible by default, by clicking the
+:icon:`oi-settings-adjust` :guilabel:`(settings)` icon and ticking the :guilabel:`Carrier` checkbox.
+
+.. tip::
+   Other useful columns to enable can be :guilabel:`Zip` code, :guilabel:`Shipping Weight` and
+   :guilabel:`Shipping Volume`.
+
+Next, tick the checkbox to far left of the delivery orders intended to go into the batch. After the
+deliveries are selected, click into one of the line's :guilabel:`Carrier` fields. In the resulting
+drop-down menu, choose the desired vehicle's :ref:`delivery method
+<inventory/shipping_receiving/delivery-method-for-carrier>`. In the resulting
+:guilabel:`Confirmation` pop-up window, click :guilabel:`Confirm`.
+
+.. figure:: dispatch/set-carrier.png
+   :alt: Set carrier.
+
+   The delivery method `Truck 1-MER-001` is set as the :guilabel:`Carrier` for two delivery orders.
+
+.. _inventory/shipping_receiving/create-batch:
+
+Create batch
+------------
+
+With the carrier set, add the orders to a batch by having the desired orders checked. Then, click
+the :icon:`fa-cog` :guilabel:`Actions` button and click either :guilabel:`Add to batch` or
+:guilabel:`Add to wave` to group the orders. In the resulting pop-up window, ensure the
+:guilabel:`Add to` field is set :guilabel:`a new [batch/wave] transfer`, and click
```

Comment:
```suggestion
With the carrier set, add the desired orders to a batch by ticking the checkbox next to each order. Then, click
the :icon:`fa-cog` :guilabel:`Actions` button and click either :guilabel:`Add to batch` or
:guilabel:`Add to wave` to group the orders. In the resulting pop-up window, ensure the
:guilabel:`Add to` field is set to :guilabel:`a new [batch/wave] transfer`, and click
```
I would add a note directly after the first sentence to explain that the ticked orders that had the carrier applies remain ticked. And that ONLY if they wanted to add more orders, they need to tick those additional boxes (but they should add the carrier). Just a thought- since they would typically be adding what they just set the carrier to. I just noticed they stayed ticked which seemed unusual, so you can expand on that a touch.

Reply:
Thank you! this is a helpful note to add